src/bidiMapper/domains/context/context.ts

```diff
@@ -108,6 +109,49 @@ export class Context {
     });
   }
 
+  private _initializeLogEntryAddedEventListener() {
+    this._cdpClient.Runtime.on('consoleAPICalled', async (params) => {
+      const args = await Promise.all(
+        params.args.map(async (arg, idx) => {
+          let argObject = (await this._serializeCdpObject(
+            arg
+          )) as CommonDataTypes.LogRemoteValue;
+          if (idx > 0) {
+            argObject.value += '\u0020';
```

Comment:
Looks like this is for the text formatting in step 5? We need to do this later for computing the value of `text` but we shouldn't modify the BiDi values in the `args` array.

Reply:
Implemented a formatter method to get preview of formatted texts in Bidi.